docs/source/howto/level1-csd.rst

```diff
@@ -0,0 +1,312 @@
+
+.. figure:: img/tutorial-l1csd-s2grid-berlin.jpg
+
+    AOI (blue polygon) in relation to the Sentinel-2 tiling grid (black lines).
+
+As we can see, the two northern tiles are probably sufficient to get all the data we need. The north/south overlap of T33UUU/T33UUT and T33UVU/T33UVT does not provide additional data, neither does the overlap of T33UUU and T33UQD. We can directly specify the desired tiles (comma-separated) as aoi:
+
+.. code-block:: none
+
+    force-level1-csd meta gcs_berlin/level1 gcs_berlin/level1/l1_pool.txt T33UUU,T33UVU -n -c 0,70 -d 20190101,20191231 -s S2B
+
+    Querying the metadata catalogue for Sentinel2 data
+    Sensor(s): S2B
+    Tile(s): T33UUU,T33UVU
+    Daterange: 20190101 to 20190430
+    Cloud cover minimum: 0%, maximum: 70%
+
+    44 Sentinel2 Level 1 scenes matching criteria found
+    21.06GB data volume found.
+
+    Done.
+
+
+When we're happy with the results, we can remove the ``-n | --no-act`` option to start downloading
+
+.. code-block:: none
+
+    force-level1-csd meta gcs_berlin/level1 gcs_berlin/level1/l1_pool.txt T33UUU,T33UVU -c 0,70 -d 20190101,20191231 -s S2B
+
+    Querying the metadata catalogue for Sentinel2 data
+    Sensor(s): S2B
+    Tile(s): T33UUU,T33UVU
+    Daterange: 20190101 to 20190430
+    Cloud cover minimum: 0%, maximum: 70%
+
+    44 Sentinel2 Level 1 scenes matching criteria found
+    21.06GB data volume found.
+
+    Starting to download 44 Sentinel2 Level 1 scenes
+
+
+    Downloading S2B_MSIL1C_20190119T101339_N0207_R022_T33UVU_20190119T121912(5 of 44)...
+
+    |===========                                                                            |  13 %
+
+
+The status bar will update after every scene that has successfully been downloaded. Any scenes that have already been downloaded will be skipped and a notification will be printed. If the download was canceled at some point and a scene wasn't fully downloaded, FORCE Level 1 CSD will delete that scene and redownload it.
+
+
+A note on Sentinel-2 scenes
+"""""""""""""""""""""""""""
+
+The processing baseline for Landsat data is standardized by the Collection format. For Sentinel-2 we found that there were duplicate scenes with different processing baselines or processing dates. FORCE Level 1 CSD will check for duplicates and only download the version with the highest processing baseline or latest processing date if the processing baseline is the same.
+
+
+Saving metadata
+---------------
+
+FORCE Level 1 CSD offers the possibility to save the metadata of the scenes that matched the specified requirements. Having easy access to this kind of information can be very helpful when creating statistics and visualizations about data availability over time, cloud cover distribution over time, data volume (e.g., per sensor or year), etc. Using the ``-k | --keep-meta`` option will save the results of the current query to the Level 1 datapool folder under the file name
```

Comment:
Say what the date is? Could be confused between -d and $TIME

Reply:
I'm sorry I don't understand...?

Reply:
The timestamp in the filename could refer to the settings in -d, or it could refer to the current time

Reply:
Oh I see! Sure thing.